### PR TITLE
Rating and toast component

### PR DESCRIPTION
**Pull request by samyukthasudhakar**
As part of this PR, the following components have been created

- Rating
- Toast

Live : https://rating-and-toast.netlify.app/docs.html#rating

### CODE REVIEW

File: docs.html [hunk 1]
```diff
@@ -758,6 +760,113 @@ <h3>Demo</h3>
...
+                    sandbox="allow-scripts allow-same-origin">
+                    </iframe>
+                </div>
+            </section>
+
+            <!--Toast Component-->
+            <section id="toast" class="component-section">
+                <div class="component-container">
+                    <h2 class="component-heading">Toast</h2>
+                    <p class="component-description">
```
Comment (abhisheksahani-nova) [inline docs.html:832]: LGTM , just one thing to make your HTML code consistent, in rating component description yu have used span tag , and in toast p tag , use only one to keep consistency

**Review by abhisheksahani-nova — APPROVED** [review]

**Review by foolhardy21 — APPROVED** [review]
LGTM. On thing - in the dynamic rating bar, you cannot decrease the stars given. If this is intentional then pls ignore.

File: docs.html (hunk 1 above)
Comment (samyukthasudhakar) [reply docs.html:832]: Sure will keep in mind and make the changes. Thank you.

**Comment by samyukthasudhakar** [comment]
> LGTM. On thing - in the dynamic rating bar, you cannot decrease the stars given. If this is intentional then pls ignore.

Yes, this was intentional. Will add this feature going forward. Thank you !

**Pull request merged**